### ISSUE TITLE
Editing a Data source URL: Fields should appear below collapsable component

### ISSUE THREAD

**Issue by sachajudd** [silverstripe/silverstripe-ckan-registry#129]
After some discussion with @clarkepaul we've decided that it might make more sense when editing a data source URL that the fields appear below the Data source title rather than above.

Quick mock up of what this might look like: 
![image](https://user-images.githubusercontent.com/24258161/51222963-ce6b9a00-19a4-11e9-9fc5-dd27015377e8.png)

To do: Currently still deciding if it would need a background colour.

**Comment by robbieaverill** [comment]
This could actually be a little tricky to achieve, because the field is a FormField and the title is a GridFieldComponent (part of a different FormField). We'd need to change the FormField into a GridFieldComponent probably, or move the title (GridFieldComponent) to be part of the FormField instead of where it is at the moment.

![image](https://user-images.githubusercontent.com/5170590/51238173-99485300-1976-11e9-95cc-0b5008b599b2.png)

**Comment by ScopeyNZ** [comment]
Yeah I think there's a search form toggle component that we might be able to leverage if we go for the latter solution (move the title to be part of a React component).
